src/parser/permit-generation-module.ts

```diff
@@ -100,13 +102,77 @@ export class PermitGenerationModule implements Module {
           config.permitRequests
         );
         result[key].permitUrl = `https://pay.ubq.fi?claim=${encodePermits(permits)}`;
+        await this._savePermitsToDatabase(result[key].userId, { issueUrl: payload.issueUrl, issueId }, permits);
       } catch (e) {
         console.error(e);
       }
     }
     return result;
   }
 
+  async _getOrCreateIssueLocation(issue: { issueId: number; issueUrl: string }) {
+    let locationId: number | null = null;
+
+    const { data: locationData } = await this._supabase
+      .from("locations")
+      .select("id")
+      .eq("issue_id", issue.issueId)
+      .eq("node_url", issue.issueUrl)
+      .single();
+
+    if (!locationData) {
+      const issueItem = await getRepo(parseGitHubUrl(issue.issueUrl));
+      const { data: newLocationData, error } = await this._supabase
+        .from("locations")
```

Comment:
Yeah we should probably stop using that table and instead just add a new URL column wherever it makes sense in the other tables.

Reply:
Used it because it is still there and we didn't update it to something different but indeed it should be changed. Might wanna have a different task for this so we can also migrate all the current values as well (there are 43 records)

Reply:
Don't worry about data migrations. Basically none of the old data is valuable. No need to do extra work to save it.